Species_dist

MGrezlik opened the pull request:
**Description**
Added all scripts needed to create the species_dist ecodata object. 

**R**
species_dist.R completes all steps formerly completed in [species_dist_from_kevin.R](https://github.com/NEFSC/READ-EDAB-CurrentConditions/blob/main/R/species_dist_from_kevin.R) and [species_dist_prep_from_kevin.R](https://github.com/NEFSC/READ-EDAB-CurrentConditions/blob/main/R/species_dist_prep_from_kevin.R)

**data-raw**
Leaving example_test_runs.R and mgrezlik_example_test_runs.R in the data-raw folder as others mentioned they would be helpful when testing their own automation.

workflow_species_dist.R defines the function

compare_nes_coast_2.pdf compares the output of old scripts to updated output. Source 'kevin' is old output. Source 'nes_coast_2' is current output and specifies that I used nes_coast_2.csv as the static input for coastline lat lon values. This is the same static coastline csv used for the old output. There were 2 additional csv files of lat lon values in the [old data-raw folder](https://github.com/NEFSC/READ-EDAB-CurrentConditions/tree/main/data-raw)

**DESCRIPTION**
Added packages used in species_dist.R to the Imports section

**still to do**
I will create a separate pull request to update template_readme.md. That file was created after this branch diverted from dev and I didn't want to create merge conflicts.

jcaracappa1 reviewed (commented):
Can you upload a comparison between the old and new indicators here too?

jcaracappa1 reviewed (approved):
Looks great.

jcaracappa1 commented:
@andybeet Any idea why these checks were failing?

MGrezlik commented:
> I think it is because dependencies are not specified in the DESCRIPTION file. All packages need to be specified there.
> 
> https://github.com/NEFSC/READ_EDAB_SOE_Workflows/actions/runs/15856229761/job/44702146949?pr=13#step:6:747

I added those to the DESCRIPTION file and updated the pull request to reflect the change. This created a small conflict. Should I resolve that on the web editor?
